### PR TITLE
Create environment_variables.md

### PR DESCRIPTION
**Pull request by thomasr8**
Created Environment Variables file and added to API Reference Guide since it was missing from guide.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: src-api/source/includes/environment_variables.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## List Environment Variables
+
+Returns a list of all environment variables.
+
+``sh
+curl https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/envvar?circle-token=:token
+``
+
+``json
```
Comment (teesloane) [inline src-api/source/includes/environment_variables.md]:
```suggestion
```json
```

File: src-api/source/includes/environment_variables.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Environment Variables
+
+## List Environment Variables
+
+Returns a list of all environment variables.
+
+``sh
```
Comment (teesloane) [inline src-api/source/includes/environment_variables.md]:
```suggestion
```sh
```

File: src-api/source/includes/environment_variables.md (hunk 1 above)
Comment (thomasr8) [reply src-api/source/includes/environment_variables.md]: fixed

File: src-api/source/includes/environment_variables.md (hunk 2 above)
Comment (thomasr8) [reply src-api/source/includes/environment_variables.md]: fixed

**Pull request merged**